backend/src/Controller/StoreController.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace App\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\Routing\Annotation\Route;
+
+use Symfony\Component\Serializer\SerializerInterface;
+use App\Entity\Store;
+use App\Repository\StoreRepository;
+
+class StoreController extends AbstractController
+{
+    /**
+     * @Route("/store", methods={"GET"})
+     */
+    public function index (
+        StoreRepository $repository,
+        SerializerInterface $serializer
+    ): JsonResponse {
```

Comment:
Keep in mind: the serialize might not work, when you have tables with relations

Reply:
Thank you i changed it now